packages/signature-controller/package.json

```diff
@@ -53,7 +53,7 @@
   },
   "peerDependencies": {
     "@metamask/approval-controller": "^5.1.2",
-    "@metamask/keyring-controller": "^12.2.0",
+    "@metamask/keyring-controller": "^13.0.0",
     "@metamask/logging-controller": "^2.0.2"
```

Comment:
Only major version change reflected in peer dependency due to https://github.com/MetaMask/core/pull/3881.